lib/galaxy/webapps/galaxy/api/tags.py

```diff
@@ -31,7 +31,7 @@ def get_tags_manager(app: StructuredApp = Depends(get_app)) -> TagsManager:
 
 
 @cbv(router)
-class FastAPITags:
+class FastAPITags(UsesTagsMixin):
```

Comment:
😆 No, not this - we need to kill that mixin not expand its use. If you're not willing to bite off refactoring that mixin to become your manager and least just inject app and grab its copy of tag_handler.

Reply:
Or - just have the manager consume ``app`` and grap the tag handler from that. This will be even cleaner down the road.

Reply:
>No, not this - we need to kill that mixin not expand its use.

Then I didn't understand it well the first time sorry😆

>Or - just have the manager consume app and grap the tag handler from that. This will be even cleaner down the road.

This brings up some pretty basic doubts I've been having about the `app` instance and its context/scope:
The `trans.app` is exactly the same instance as the `app` that gets injected in the controller? Can they be used interchangeably? or the `trans.app` is more specific to the current user making the request?

Reply:
I could be wrong, but I think app is the same. I think trans is what is different and what it does with the thread bound database session.

Reply:
Same here, it is my understanding that app == trans.app, at least in this context. I used this to get rid of one or the other at some refactoring step in the configuration controller.

Reply:
Thanks, I will get rid of one of them then :)